src/components/sections/Incentives/IncentivesHeader.tsx

```diff
@@ -1,37 +1,19 @@
-import Icon from 'src/components/ui/Icon'
-
 import Incentives from './Incentives'
 import Section from '../Section'
 
-const incentives = [
-  {
-    icon: <Icon name="Truck" width={32} height={32} />,
-    title: 'Buy online',
-    firstLineText: 'Get Free Shipping',
-  },
-  {
-    icon: <Icon name="Calendar" width={32} height={32} />,
-    title: 'Free return',
-    firstLineText: '30 days to return',
-  },
-  {
-    icon: <Icon name="Gift" width={32} height={32} />,
-    title: 'Gift cards',
-    firstLineText: '$20 / $30 / $50',
-  },
-  {
-    icon: <Icon name="Storefront" width={32} height={32} />,
-    title: 'Physical Stores',
-    firstLineText: '+40 Stores in Brazil',
-  },
-  {
-    icon: <Icon name="ShieldCheck" width={32} height={32} />,
-    title: 'Buy online',
-    firstLineText: 'Get Free Shipping',
-  },
-]
+interface Incentive {
+  icon: string
+  title?: string
```

Comment:
I think `title` should be optional, since we had `IncentivesFooter` with `Incentives` without title (as a variation) in the beginning. I'm going to change `title` in `storeframework` to be optional (just in case).